core-api/src/entities/permission.entity.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    Column,
+    Entity,
+    PrimaryGeneratedColumn,
+    CreateDateColumn,
+    UpdateDateColumn,
+} from "typeorm";
+import { Length } from "class-validator";
+import Messages from "../utils/messages";
+
+/**
+ * @namespace Entities
+ * @class User
+ */
+@Entity()
+export class Permisson {
+    @PrimaryGeneratedColumn()
+    private id: number;
+
+    @Column({ length: 64 })
+    @Length(3, 20, {
```

Comment:
I believe the second parameter means maximum length, shouldn't it be 64 instead of 20?

Reply:
I believe the second parameter actually is the maximum length of the validation message.

Reply:
Yes, but I don't believe that any permission name will get so far. We only put 64 into database for safety.

Reply:
Also if we wanted in the future to increase the length until 64, we don't need to change any scheme in database.